### PR TITLE
move sentences to initiate once

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Review by MaxBaeten — APPROVED** [review]
Good work. also travis!

File: src/emergency_speakup.cpp [hunk 1]
```diff
@@ -60,6 +45,25 @@ int main(int argc, char **argv)
...
+    sentences_pressed.push_back("Man is a robot with defects");
+
+    sentences_released.push_back("Let's move on");
+    sentences_released.push_back("Here we go again");
+    sentences_released.push_back("I'll try to do better this time");
+    sentences_released.push_back("My emergency button is released again");
+    sentences_released.push_back("Lets make this fun");
+    sentences_released.push_back("Lets get ready to rumble");
+    sentences_released.push_back("You will respect my autonomy!");
+
```
Comment (Rayman) [inline src/emergency_speakup.cpp:66]: space vs tabs issue?

Comment (MatthijsBurgh) [reply src/emergency_speakup.cpp:66]: These are two different queues, so no issue at all

Comment (Rayman) [reply src/emergency_speakup.cpp:66]: I mean that the sentences are indented with 4 spaces while the rest of the code uses tabs

Comment (MatthijsBurgh) [reply src/emergency_speakup.cpp:66]: Now I see the issue. I see that also the old code wasn't consistent. I will create a new PR to fix this.

**Pull request merged**